website/docs/seo.mdx

```diff
@@ -74,6 +112,17 @@ export default function page() {
     <Layout title="Page" description="A React page demo">
       <Head>
         <meta property="og:image" content="image.png" />
+        <meta name="twitter:card" content="summary_large_image" />
```

Comment:
Since this is generally about improving SEO docs.  And `front matter` is a pretty big deal for this, we should also in this same PR try to clean up some missing links about `front matter`.

One thing I noticed above on line 61 is the mention of `front matter`.  That should probably be turned into a link to the page on front matter.  In general, I've noticed we are pretty bad at just typing names, when they really should be links to their respective pages, especially anything like plugin, api, etc.

I think the general link for the feature would be appropriate here I think:
https://docusaurus.io/docs/markdown-features#front-matter

But I also notice we don't have a link to the API when we describe front matter on that feature section! Which should have a link to the API for the plugin, right? https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#markdown-front-matter

But we DO have an API link on the [Create a Doc](https://docusaurus.io/docs/create-doc#doc-front-matter) section about **Doc front matter**